exporters/trace/zipkin/model.go

```diff
@@ -146,10 +146,13 @@ var extraZipkinTags = []string{
 }
 
 func toZipkinTags(data *export.SpanSnapshot) map[string]string {
-	m := make(map[string]string, len(data.Attributes)+len(extraZipkinTags))
+	m := make(map[string]string, len(data.Attributes)+len(data.Resource.Attributes())+len(extraZipkinTags))
 	for _, kv := range data.Attributes {
 		m[(string)(kv.Key)] = kv.Value.Emit()
 	}
+	for _, kv := range data.Resource.Attributes() {
```

Comment:
Should this happen before adding values from `data.Attributes`?  I think we expect that `Resource` values will underlay user-added `Attribute` values.

Reply:
Done.

Reply:
I reverted this change, since adding `Resource` to zipkin tags is not mentioned in the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#summary).

> TBD : This is work in progress document and it is currently doesn't specify mapping for these fields:
> 
> OpenTelemetry fields:
> 
> Resource attributes